apps/laboratory/tests/social.spec.ts

```diff
@@ -1,30 +0,0 @@
-/**
```

Comment:
For none of the social providers, we don't have a proper solution to run this. Instead of ignoring this for every client, just wanted to remove and we could add it back once we have a solution. Otherwise it's nothing than extra code. (Might be good to see it there to be completed but idk how much it'll go)